.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .idea
-lib/**/*
\ No newline at end of file
+lib/**/*
+.secrets
```

Comment:
Consider whether adding '.secrets' to .gitignore is necessary. If this file is meant to store sensitive information, ensure that it is not accidentally committed to the repository. Verify that this aligns with your project's security practices.